ext/opcache/jit/zend_jit_arm64.dasc

```diff
@@ -10950,77 +10935,88 @@ static int zend_jit_return(dasm_State **Dst, const zend_op *opline, const zend_o
 		}
 	}
 
-	if (return_value_used == 0) {
-		|9:
-		return 1;
-	}
-
-	if (opline->op1_type == IS_CONST) {
-		zval *zv = RT_CONSTANT(opline, opline->op1);
-		|	ZVAL_COPY_CONST ret_addr, MAY_BE_ANY, MAY_BE_ANY, zv, ZREG_REG0, ZREG_TMP1, ZREG_FPR0
-		if (Z_REFCOUNTED_P(zv)) {
-			|	ADDREF_CONST zv, REG0, TMP1
-		}
-	} else if (opline->op1_type == IS_TMP_VAR) {
-		|	ZVAL_COPY_VALUE ret_addr, MAY_BE_ANY, op1_addr, op1_info, ZREG_REG0, ZREG_REG2, ZREG_TMP1, ZREG_TMP2, ZREG_FPR0
-	} else if (opline->op1_type == IS_CV) {
-		if (op1_info & MAY_BE_REF) {
-			|	LOAD_ZVAL_ADDR REG0, op1_addr
-			|	ZVAL_DEREF REG0, op1_info, TMP1w
-			op1_addr = ZEND_ADDR_MEM_ZVAL(ZREG_REG0, 0);
-		}
-		|	ZVAL_COPY_VALUE ret_addr, MAY_BE_ANY, op1_addr, op1_info, ZREG_REG0, ZREG_REG2, ZREG_TMP1, ZREG_TMP2, ZREG_FPR0
-		if (op1_info & (MAY_BE_STRING|MAY_BE_ARRAY|MAY_BE_OBJECT|MAY_BE_RESOURCE)) {
-			if (JIT_G(trigger) != ZEND_JIT_ON_HOT_TRACE ||
-			    (op1_info & (MAY_BE_REF|MAY_BE_OBJECT)) ||
-			    !op_array->function_name) {
-				|	TRY_ADDREF op1_info, REG0w, REG2, TMP1w
-			} else if (return_value_used != 1) {
-				|	// if (EXPECTED(!(EX_CALL_INFO() & ZEND_CALL_CODE))) ZVAL_NULL(retval_ptr);
-				|	SET_ZVAL_TYPE_INFO op1_addr, IS_NULL, TMP1w, TMP2
+	if (return_value_used != 0) {
+		if (opline->op1_type == IS_CONST) {
```

Comment:
Can you please try to minimize the patch, avoiding indentation changes. e.g.

```c
	if (return_value_used == 0) {
		/* pass */
	} else if (opline->op1_type == IS_CONST) {
  ```

Reply:
Yep, done